templates/debug.hbs

```diff
@@ -1,19 +1,42 @@
 <style>
     svg:first-child {
-        display: none;
+        display:none;
```

Comment:
There's still a lot of spaces missing between colons and values.

https://github.com/Lorti/grunt-svg-symbols/pull/11#discussion_r158245697